cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -139,6 +140,36 @@ func reloadLoop(
 			}
 		}(ctx)
 
+		if envConfigPath, err := getEnvConfigPath(*fConfig, *fEnvConfig); err == nil {
+			// Reloads environment variables when file is changed
+			go func(ctx context.Context, envConfigPath string) {
+				var previousModTime time.Time
+				ticker := time.NewTicker(30 * time.Second)
```

Comment:
It is hard to know if every 30 seconds is the right polling frequency.
I saw this and wondered if we could or should poll every 60 seconds.
Then I wondered if it matters.
I am assuming calling `os.Stat()` is pretty cheap so 30 seconds is fine.
I just wanted to bring some attention to it in case others had opinions...